rosbag2_test_common/include/rosbag2_test_common/process_execution_helpers_windows.hpp

```diff
@@ -104,8 +105,10 @@ ProcessHandle start_execution(const std::string & command)
   return process;
 }
 
-void stop_execution(const ProcessHandle & handle)
+void stop_execution(const ProcessHandle & handle, int signum = SIGINT)
```

Comment:
The `signum` parameter appears to be unused here. Is this intentional? The comment below implies it might be, but I just want to be sure.

Is it possible to use something like `int /*signum*/` here? Or does C++ prevent that in this case?

Reply:
the parameter is optional, hence the default value, but we can't declare a default value without a name. That's why it couldn't be commented out here. Good eye though, I also wanted to comment it out.

Reply:
Given that this is a backport though - ideally we don't really do the full review process here, instead just noting "yes this is a backport, and it isn't doing anything crazy on the merge"